tests/unit/test_refresh_utils.py

```diff
@@ -287,19 +289,19 @@ async def test_is_valid_with_expired_metadata() -> None:
     assert not await _is_valid(task)
 
 
-def test_downscope_credentials_service_account(fake_credentials: Credentials) -> None:
-    """
-    Test _downscope_credentials with google.oauth2.service_account.Credentials
-    which mimics an authenticated service account.
-    """
-    # set all credentials to valid to skip refreshing credentials
-    with patch.object(Credentials, "valid", True):
-        credentials = _downscope_credentials(fake_credentials)
-    # verify default credential scopes have not been altered
-    assert fake_credentials.scopes == SCOPES
-    # verify downscoped credentials have new scope
-    assert credentials.scopes == ["https://www.googleapis.com/auth/sqlservice.login"]
-    assert credentials != fake_credentials
+# def test_downscope_credentials_service_account(fake_credentials: Credentials) -> None:
```

Comment:
Let's link to the issue to fix this test.

Reply:
Done, will add a comment: https://github.com/GoogleCloudPlatform/cloud-sql-python-connector/issues/901